pkg/graph/schema.resolvers.go

```diff
@@ -512,6 +528,20 @@ func (r *mutationResolver) UpdateAccessibilityRequestStatus(ctx context.Context,
 		return nil, err
 	}
 
+	if latestStatusRecord.Status != input.Status {
```

Comment:
I thought we had descoped this check for now in favor of changing the form so that you can only "save" as new status. But I might be misremembering that...

Reply:
Eh, it's so straightforward I may as well leave it